src/bot/events/interactionCreate.js

```diff
@@ -38,13 +39,18 @@ module.exports = {
         await userCtx(interaction);
       } else if (interaction.isCommand() || interaction.isAutocomplete()) {
         const path = await getPath(interaction);
-        const command = interaction.client.commands.get(path);
+        const command = interaction.client.commands.get(path) ?? interaction.client.adminCommands.get(interaction.commandName);
         if (!command)
           return console.log('No command found: ', path);
 
+        if (command.isAdmin && userPrivileges[interaction.user.id] < command.requiredPrivileges) {
```

Comment:
Maybe add  '&& userPrivileges[interaction.user.id]', check for it to be defined, so it doesnt make a compare 'undefined < command.requiredPrivileges'